nixos/doc/manual/default.nix

```diff
@@ -1,27 +1,27 @@
 { pkgs, options, version, revision, extraSources ? [] }:
 
 with pkgs;
-with pkgs.lib;
```

Comment:
For generating the epub the commandline tool `zip` is required. When I add it to `buildInputs` while keeping the global `with pkgs.lib;` statement  I get the following error message: `trace: lib.zip is deprecated, use lib.zipAttrsWith instead`.